src/AutoRest.CSharp.V3/CodeGen/CodeWriter.cs

```diff
@@ -248,6 +254,13 @@ public string ToFormattedCode()
 
         public override string? ToString() => _builder.ToString();
 
+        public static string Materialize(CodeWriterDelegate writer)
```

Comment:
This method is temporary. When we'll cleanup all string formatting in JSON serializer writer it can be removed.